sdk/ai/azure-ai-generative/azure/ai/generative/_user_agent.py

```diff
@@ -0,0 +1,6 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+from azure.ai.generative._version import VERSION
+
+USER_AGENT = "{}/{}".format("azure-ai-generative", VERSION)
```

Comment:
Was this generated? the user-agent should follow the format:

{azsdk-<sdk_language>-<package_name>/<package_version> }+<platform_info>

Reply:
This was created manually. Should I change it to your format?

Reply:
This is the format that we use per the [SDK guidelines](https://azure.github.io/azure-sdk/general_azurecore.html). I see that we use the `"{}/{}".format("azure-ai-ml", VERSION)` format in azure-ai-ml, too. I don't have the context of whether we intentionally use a different format for these packages. If we don't want to change it right now, maybe just open an issue so we don't get forget about it

Reply:
Sure, I'll add a task in our backlog for it!